dialogflow/spec/context_spec.rb

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rspec"
+require "google/cloud/dialogflow"
+
+require_relative "../context_management"
+require_relative "../detect_intent_texts"
```

Comment:
Why does the Context Management spec require the Detect Intents Texts sample? Typo? Leftover from old code?

Reply:
oh, I see you're using it to "create a session" ...